tests/framework/e2e.go

```diff
@@ -16,14 +16,17 @@ package framework
 
 import (
 	"context"
-	"os"
-	"testing"
-
 	"go.etcd.io/etcd/client/pkg/v3/testutil"
 	"go.etcd.io/etcd/tests/v3/framework/config"
 	"go.etcd.io/etcd/tests/v3/framework/e2e"
+	"os"
+	"strings"
+	"testing"
+	"time"
```

Comment:
Please put the golang build-in packages in the first group (immediately follow `context` (line 18))